content/synthetics/faq/uptime-check-internal-website.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Create an Uptime Check for private website
+kind: faq
+beta: true
+further_reading:
+- link: "synthetics/"
+  tag: "Documentation"
+  text: "Manage your checks"
+- link: "synthetics/"
+  tag: "Documentation"
+  text: "Manage your checks"
+---
+
+<div class="alert alert-warning">Synthetics is in beta for the Datadog US Region. To request access, complete the <a href="https://app.datadoghq.com/synthetics/beta">Datadog Synthetics request form</a>.</div>
+
+Monitor your internal staging/intranet environment by appending authentication credentials in the **Advanced options** of your Uptime Check to satisfy security concerns.
```

Comment:
Maybe you can precise that secrets will be fully encrypted all along to the request. This could be a question some may ask to the support.